### ISSUE TITLE
[Bug] Upgrading will result in loss of rewards？

### ISSUE THREAD

**Issue by mrbaiwei** [Chia-Network/chia-blockchain#15214]
### What happened?

I upgraded from 1.5.1 to the latest version 1.8.0, but I still can't get rewards, or the rewards are very low. the same happened when I upgraded to 1.7.1. Check that the synchronization delay is normal, and the challenge data is all normal
![image](https://user-images.githubusercontent.com/16459219/236435334-5906da53-74ed-472a-84e9-07930578b3cc.png)

# plot info
Plot count for all harvesters: 1821037
Total size of plots: 176.015 PiB
Estimated network space: 23.091 EiB
Expected time to win: 41 minutes

### Version

1.8.0

### What platform are you using?

Windows

### What ui mode are you using?

GUI

### Relevant log output

```shell
#1.8.0
2023-05-05T14:19:59.441 harvester chia.harvester.harvester: INFO     12 plots were eligible for farming 100f89d053... Found 0 proofs. Time: 0.59407 s. Total 5110 plots
2023-05-05T14:20:08.179 harvester chia.harvester.harvester: INFO     13 plots were eligible for farming 100f89d053... Found 0 proofs. Time: 0.53973 s. Total 5110 plots
2023-05-05T14:20:17.246 harvester chia.harvester.harvester: INFO     11 plots were eligible for farming 100f89d053... Found 0 proofs. Time: 0.58606 s. Total 5110 plots
2023-05-05T14:20:25.971 harvester chia.harvester.harvester: INFO     10 plots were eligible for farming 100f89d053... Found 0 proofs. Time: 0.47227 s. Total 5110 plots
2023-05-05T14:20:34.911 harvester chia.harvester.harvester: INFO     16 plots were eligible for farming 100f89d053... Found 0 proofs. Time: 0.58283 s. Total 5110 plots
2023-05-05T14:20:46.685 harvester chia.harvester.harvester: INFO     14 plots were eligible for farming 100f89d053... Found 0 proofs. Time: 0.60244 s. Total 5110 plots
2023-05-05T14:20:55.327 harvester chia.harvester.harvester: INFO     11 plots were eligible for farming 100f89d053... Found 0 proofs. Time: 0.49840 s. Total 5110 plots
2023-05-05T14:21:03.320 harvester chia.harvester.harvester: INFO     7 plots were eligible for farming 100f89d053... Found 0 proofs. Time: 0.62304 s. Total 5110 plots
2023-05-05T14:21:13.650 harvester chia.harvester.harvester: INFO     16 plots were eligible for farming 100f89d053... Found 0 proofs. Time: 1.05901 s. Total 5110 plots
2023-05-05T14:21:21.858 harvester chia.harvester.harvester: INFO     10 plots were eligible for farming 100f89d053... Found 0 proofs. Time: 0.49213 s. Total 5110 plots
2023-05-05T14:21:30.487 harvester chia.harvester.harvester: INFO     7 plots were eligible for farming 100f89d053... Found 0 proofs. Time: 0.51077 s. Total 5110 plots
2023-05-05T14:21:40.849 harvester chia.harvester.harvester: INFO     14 plots were eligible for farming 100f89d053... Found 0 proofs. Time: 0.53056 s. Total 5110 plots
2023-05-05T14:21:49.334 harvester chia.harvester.harvester: INFO     16 plots were eligible for farming 100f89d053... Found 0 proofs. Time: 0.70493 s. Total 5110 plots

#1.5.1
2023-05-05T16:27:49.779 harvester chia.harvester.harvester: INFO     8 plots were eligible for farming b4eca28101... Found 0 proofs. Time: 0.30442 s. Total 5110 plots
2023-05-05T16:27:58.842 harvester chia.harvester.harvester: INFO     14 plots were eligible for farming b4eca28101... Found 0 proofs. Time: 0.43753 s. Total 5110 plots
2023-05-05T16:28:07.531 harvester chia.harvester.harvester: INFO     4 plots were eligible for farming 05b1ea354b... Found 0 proofs. Time: 0.30590 s. Total 5110 plots
2023-05-05T16:28:20.396 harvester chia.harvester.harvester: INFO     8 plots were eligible for farming 05b1ea354b... Found 0 proofs. Time: 0.53123 s. Total 5110 plots
2023-05-05T16:28:28.619 harvester chia.harvester.harvester: INFO     11 plots were eligible for farming 05b1ea354b... Found 0 proofs. Time: 0.38749 s. Total 5110 plots
2023-05-05T16:28:38.563 harvester chia.harvester.harvester: INFO     10 plots were eligible for farming 05b1ea354b... Found 0 proofs. Time: 0.31812 s. Total 5110 plots
2023-05-05T16:28:47.135 harvester chia.harvester.harvester: INFO     12 plots were eligible for farming 05b1ea354b... Found 0 proofs. Time: 0.39459 s. Total 5110 plots
2023-05-05T16:28:56.104 harvester chia.harvester.harvester: INFO     14 plots were eligible for farming 05b1ea354b... Found 0 proofs. Time: 0.44242 s. Total 5110 plots
2023-05-05T16:29:05.048 harvester chia.harvester.harvester: INFO     11 plots were eligible for farming 05b1ea354b... Found 0 proofs. Time: 0.43825 s. Total 5110 plots
2023-05-05T16:29:13.733 harvester chia.harvester.harvester: INFO     10 plots were eligible for farming 05b1ea354b... Found 0 proofs. Time: 0.29278 s. Total 5110 plots
2023-05-05T16:29:26.237 harvester chia.harvester.harvester: INFO     6 plots were eligible for farming 05b1ea354b... Found 0 proofs. Time: 0.39406 s. Total 5110 plots
2023-05-05T16:29:36.430 harvester chia.harvester.harvester: INFO     7 plots were eligible for farming 05b1ea354b... Found 0 proofs. Time: 0.34388 s. Total 5110 plots
```

**Comment by DrakeFS** [comment]
This is far more likely due to "luck" rather than a bug.   It is not uncommon for a farmer to see 8x or more/less effort required for a block win.

Also, posting 3 minutes worth of plots passing the filter will not show an issue.

**Comment by MumfMeisterT** [comment]
As to @DrakeFS's' point, we do not see an issue here, this does seem to be a luck issue. 
We would recommend updating all of your harvesters and see if your times are looking closer to what you're expecting. If you go more than ~5 hours without a win, then there might be an issue, and we can look into it further.

**Comment by la-mikkamakka** [comment]
Hi guys, my harvester hasn't got any rewards since May 8th. 4 days straight now. I have 4045 plots, usually get 0.01 chia per day, has worked like that since started I started it up. Daily 2 rewards are not happening anymore. Issue happened on v1.61, now upgraded to 1.8. Not seeing any change. Is there a reason why rewards would stop completely? Is this just question of luck? Or a dry spell? Any thoughts?

**Comment by DrakeFS** [comment]
>  I have 4045 plots, usually get 0.01 chia per day

At 4045 plots you have a ~13 day ETW.  With variance (luck) you could see ~3+ months between block wins.  However, you state that you normally get .01 XCH per day, does this mean you are pooling?  

If you are pooling, you should be seeing pool payouts still.  If you are not, I would suggest contacting your pools support.

**Comment by la-mikkamakka** [comment]
Upgraded to 1.8, after a few days started to see rewards but 50% less then before. So to recap, v1,6 stopped working on May 8th,
no rewards until upgraded to v1.8. Now getting rewards but 50% less as it was a week before. Not pooling with others, doing solo and group NFT. There was a period last year when rewards were less then usual for a few months but it went up again. That drop was gradual, not sudden. This one felt different, like system was tweaked and got less chance now then before. Will keep an eye on the situation and will report back.

**Comment by MumfMeisterT** [comment]
We recommend trying to pool a portion of your plots to verify your partials and receive extensive data back on whether you are meeting your ETWs. We do not believe that there is an issue with 1.8.0 because netspace is rising continuously.

**Comment by lukeisbackontrack** [comment]
I run solo farm 1.53PiB I used to received rewards every 3 days or so, since installed 1.8.0 I haven’t received a reward. Now running 1.8.1. Last time I received a reward was 7th of May. There aren’t any errors in the log apart from drive related warning that specific directory doesn’t exist. I first didn’t make much of it and thought it’s due to bad luck, but it’s end of May and I didn’t receive a reward since 7th May. I farm solo, using GUI. Looking at chia explorer my full node is up to date. I don’t have any other choice, I  will have to pull the plug, electricity bill too expensive.

**Comment by HuoHua2019** [comment]
Yeah, sometime ago I upgraded to 1.8.1 and continue to solo. Since I found I haven't got any rewards for 12 days(usually less than 8 days), I gave it a check and found nothing but "Found 0 proofs" issues continuesly. Then I changed to pool and everything went on the correct way. Today I changed to solo again and found the same issue. So I guess something must went wrong to someone who solo only, but those who choose pool are not affected.
